indico_zodbimport/modules/badges_posters.py

```diff
@@ -63,9 +139,9 @@ def _sane_float(num):
     'right_margin': ('__BadgePDFOptions_rightMargin', float),
     'margin_columns': ('__BadgePDFOptions_marginColumns', float),
     'margin_rows': ('__BadgePDFOptions_marginRows', float),
-    'page_size': ('_pageSize', unicode),
-    'landscape': ('_landscape', bool),
-    'draw_dashed_rectangles': ('_drawDashedRectangles', bool)
+    'page_size': ('_pageSize', lambda s: getattr(PageSize, s.title())),
+    'page_orientation': ('_landscape', lambda l: PageOrientation.landscape if l else PageOrientation.portrait),
```

Comment:
... which is a silly rule IMHO. And apparently the pep8 tool doesn't enforce it.